conanfile.py

```diff
@@ -10,9 +10,16 @@
 from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
 from conan.tools.files import copy
 
-import os, sys, platform
+import os, sys, platform, traceback
```

Comment:
Are we still using traceback or can this be removed now that we're done debugging?